moddingway/services/exile_service.py

```diff
@@ -143,6 +168,25 @@ async def unexile_user(
         exiles_database.update_exile_status(exile.exile_id, ExileStatus.UNEXILED)
         logging_embed.set_footer(text=f"Exile ID: {exile.exile_id}")
 
+    # check for any sticky roles to restore
+    try:
+        roles_to_restore: list[Role] = []
+        for role in roles_database.get_sticky_roles(db_user.user_id):
+            discord_role = user.guild.get_role(int(role))
+            if discord_role is not None:
+                roles_to_restore.append(discord_role)
```

Comment:
maybe i'm reading this wrong, but wont `user.guild.get_role` always return `None` as they won't have the role?

Reply:
`user.guild` accesses the guild associated with the Member object so I'm just checking if the role exists in the guild before adding it to the list

Reply:
can we add an else statement to log that the role wasn't found in the server?

Reply:
Er sorry I meant add it to the if and output to the logger.error